src/jquery.smoothState.js

```diff
@@ -505,7 +505,30 @@
             window.location = url;
           }
         },
+        /**Reload the page */
```

Comment:
This probably needs a docblock like all the others, just to keep to coding standards.